src/config/interfaces.ts

```diff
@@ -1,12 +1,18 @@
 import { FRCallbackFactory } from '../fr-auth/callbacks/factory';
 import { Tokens } from '../shared/interfaces';
 
+interface Action {
+  type: string;
+  payload: any;
```

Comment:
If payload is an object then object type may work better here

Reply:
Payload could literally be anything: a string, a number, an object, an array ...

Reply:
Okay all good then.